docs/source/examples/semantic_segmentation.rst

```diff
@@ -0,0 +1,234 @@
+
+.. role:: hidden
+    :class: hidden-section
+
+Semantic segmentation using Poutyne
+***********************************
+
+.. note::
+
+    - See the notebook `here <https://github.com/mohammad-brdrn/Semantic_segmentation_with_Poutyne/blob/main/Semantic_segmentation.ipynb>`_
+    - Run in `Google Colab <https://drive.google.com/file/d/1Slep8z8yBwKuXO-c8-JbEHBglSjPY7lg/view?usp=sharing>`_
+
+Semantic segmentation refers to the process of linking each pixel in an image to a class label. We can think of semantic segmentation as image classification at a pixel level. The image below clarifies the definition of semantic segmentation.
+
+.. image:: /_static/img/semantic_segmentation/semantic_segmentation.png
```

Comment:
Is there a source for this image? If so, we should link it. Also, was there a source for the image in the other notebook?